exec/session.go

```diff
@@ -364,3 +377,16 @@ func (r *Result) open() sliceio.ReadCloser {
 	}
 	return sliceio.MultiReader(readers...)
 }
+
+func writeTraceFile(tracer *tracer, path string) {
+	w, err := os.Create(path)
```

Comment:
Should we close this file? I don't know what Go guarantees about closing files on program shutdown, but it seems better to close, or at least document that it doesn't.